apps/admin-gui/src/app/admin/pages/admin-page/admin-searcher/admin-searcher.component.html

```diff
@@ -0,0 +1,86 @@
+<h1 class="page-subtitle">{{'ADMIN.SEARCHER.TITLE' | translate}}</h1>
+<mat-tab-group (selectedIndexChange)="tabChanged($event)">
+  <mat-tab class="mb-2">
+    <ng-template matTabLabel>
+      {{'ADMIN.SEARCHER.TAB_USERS' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <perun-web-apps-attribute-search-select
+        *ngIf="!loading"
+        [attributes]="availableAttrDefs"
+        (search)="searchEntities($event, 'user')">
+      </perun-web-apps-attribute-search-select>
+      <app-users-list
+        *ngIf="!loadingEntityData"
+        [tableId]="tableId"
+        [displayedColumns]="['user', 'id', 'name']"
+        [users]="entities">
+      </app-users-list>
+    </ng-template>
+  </mat-tab>
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'ADMIN.SEARCHER.TAB_MEMBERS' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <div class="mt-3">
+        <perun-web-apps-vo-search-select
+          *ngIf="!loadingEntityData"
+          [vos]="vos"
+          [vo]="selectedVo"
+          (voSelected)="voSelected($event)">
+        </perun-web-apps-vo-search-select>
+      </div>
+      <perun-web-apps-attribute-search-select
+        *ngIf="!loading"
+        [attributes]="availableAttrDefs"
+        (search)="searchEntities($event, 'member')">
+      </perun-web-apps-attribute-search-select>
+      <perun-web-apps-members-list
+        *ngIf="!loadingEntityData"
+        [tableId]="tableId"
+        [displayedColumns]="['id', 'voId', 'userId', 'status']"
```

Comment:
add name column :)

Reply:
As I mentioned earlier, we have just a `Member` object instead of a `RichMember` object at this moment, so we can't display the user name for these members. But in the old GUI, there isn't a name column too, so it shouldn't be a problem.
In the future, we can extend the methods for Rich/Enriched entities and display more columns.